expense-backend/src/index.js

```diff
@@ -0,0 +1,24 @@
+// Importing 'express' module and 'dotenv' module
+const express = require('express');
+const dotenv = require('dotenv');
+
+dotenv.config({ path: 'custom.env' });
```

Comment:
Don't use a custom.env, use the simple one.

Reply:
Ok Sir, i'll change it

Reply:
Please Sir @stephane-segning is it proper to push  `.env` file ?

Reply:
No. It'll stay local. This should be documented for every developer to use at create one such file before he starts working properly

Reply:
Ok Sir, noted.